_posts/2022-07-26-mfa-ui-only-removed.md

```diff
@@ -0,0 +1,19 @@
+---
+title: An Update to RubyGems MFA Levels
+layout: post
+author: Kevin Lin
+author_email: kevin.lin@shopify.com
+---
+
+# Summary
+
+The `UI only` multi-factor authentication level is being removed.
+Users who are currently on the `UI only` MFA level will still remain on that level, but will be migrated to the `UI and gem signin` level on August 26th, 2022.
```

Comment:
Let push this to 8 Sept. Please update the date of this post to 8 Aug as well.

Reply:
Sorry I missed this, I'll assume that a month of buffer is still desired. I'll put it on the 22nd, next monday.